concepticondata/references/references.bib

```diff
@@ -3102,3 +3102,78 @@ @misc{Hsiu2015
   doi = {10.5281/zenodo.1127804},
   publisher={Center for Research in Computational Linguistics}
 }
+
+@article{Maciejewski2016,
+  title={{Relative meaning frequencies for 100 homonyms: British eDom norms}},
+  author={Maciejewski, Greg and Klepousniotou, Ekaterini},
+  journal={Journal of Open Psychology Data},
+  volume={4},
+  year={2016},
+  publisher={Ubiquity Press}
+}
+
+@misc{Parks1998,
+  title={Wordsmyth English Dictionary-Thesaurus. [Electronic version]},
+  author={Parks, R and Ray, J and Bland, S},
+  year={1998},
+  publisher={University of Chicago},
+  url = {http://www.wordsmyth.net}
+}
+
+@article{Armstrong2012,
+  title={{eDom: Norming software and relative meaning frequencies for 544 English homonyms}},
+  author={Armstrong, Blair C. and Tokowicz, Natasha and Plaut, David C.},
+  journal={Behavior Research Methods},
+  volume={44},
+  number={4},
+  pages={1015--1027},
+  year={2012},
+  publisher={Springer}
+}
+
+@article{BNC2007,
+  title={{The British National Corpus, Version 3}},
+  author={BNC Consortium},
+  journal={Distributed by Oxford University Computing Services},
+  year={2007}
+}
+
+@book{Fellbaum1998,
+  title={WordNet Dictionary 2.1},
+  author={Fellbaum, Christiane},
+  year={1998},
+  publisher={MIT Press}
+}
+
+@article{Hoffman2013,
+  title={Semantic diversity: A measure of semantic ambiguity based on variability in the contextual usage of words},
+  author={Hoffman, Paul and Ralph, Matthew A. Lambon and Rogers, Timothy T.},
+  journal={Behavior Research Methods},
+  volume={45},
+  number={3},
+  pages={718--730},
+  year={2013},
+  publisher={Springer}
+}
+
+@article{Stadthagen2006,
+  title={The Bristol norms for age of acquisition, imageability, and familiarity},
+  author={Stadthagen-Gonzalez, Hans and Davis, Colin J.},
+  journal={Behavior Research Methods},
+  volume={38},
+  number={4},
+  pages={598--605},
+  year={2006},
+  publisher={Springer}
+}
+
+@article{Coltheart1981,
+  title={The {MRC} psycholinguistic database},
+  author={Coltheart, Max},
+  journal={The Quarterly Journal of Experimental Psychology Section A},
+  volume={33},
+  number={4},
+  pages={497--505},
+  year={1981},
+  publisher={SAGE Publications Sage UK: London, England}
```

Comment:
Publisher is publisher, it should never be mixed with the `address`, but you don't need the publisher for journals anyway.